mlir/tools/mlir-miopen-lib/mlir-miopen-lib.h

```diff
@@ -0,0 +1,85 @@
+//===-------------- mlir-miopen-lib.h - C API ---------------------*- C -*-===//
```

Comment:
[Recommendation] Rename file to mlirmiopen.h

Reply:
Will probably go with `Mlirmiopen.h` to be consistent with other C APIs.

Reply:
I recommend lowercase.h

Reply:
Yea, lowercase for filenames are better, as this avoids potential issues with using the repo on case-insensitive filesystems.

Reply:
No offense guys, I'd personally prefer lower case that's why how this file is named as-is. However, I did take a complete look at the naming convention in all other places and they are uniformly in Upper Camel Case. 
  - Includes in the dialect folder, [here](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/tree/miopen-dialect/mlir/include/mlir/Dialect/Affine/IR).
  - Includes in the C-API folder, [here](https://github.com/ROCmSoftwarePlatform/llvm-project-mlir/tree/miopen-dialect/mlir/include/mlir-c)

Since this is an exported C-API, I have no choice but to name it with `Miir.h` to follow the naming convention in LLVM/MLIR.